src/app/view/createLogo.js

```diff
@@ -0,0 +1,19 @@
+import logo from '/static/assets/img/logo.png'
+
+export const createLogo = () => {
+  const img = document.createElement('img');
+  console.log(img);
+  img.classList.add('.header__game-logo__img');
+  console.log(img);
+  img.src = logo;
+  console.log(img);
```

Comment:
You can use our elementCreator function:
`const img = elementCreator({....`
Remove console.log lines